src/codegen/sys/lib_.rs

```diff
@@ -18,9 +19,12 @@ pub fn generate(env: &Env) {
 
     let path =  PathBuf::from(&env.config.target_path)
         .join(file_name_sys("lib"));
-    println!("Generating file {:?}", path);
 
-    save_to_file(path, &mut |w| generate_lib(w, env));
+    let parent = path.parent().unwrap();
+    fs::create_dir(parent).ok();
```

Comment:
Haha, this is a novel way of ignoring errors. `let _ = ...` might be more obvious.